test/src/elementhandle.spec.ts

```
@@ -572,10 +615,16 @@ describe('ElementHandle specs', function () {
       );
       puppeteer.registerCustomQueryHandler('getByClass', {
         queryOne: (element, selector) => {
-          return element.querySelector(`.${selector}`);
+          if (element instanceof Element || element instanceof Document) {
+            return element.querySelector(`.${selector}`);
+          }
+          return null;
         },
         queryAll: (element, selector) => {
-          return element.querySelectorAll(`.${selector}`);
+          if (element instanceof Element || element instanceof Document) {
+            return element.querySelectorAll(`.${selector}`);
+          }
+          return [];
```

let's throw a runtime error when the element type is not right?